### PR TITLE
cronv2 should require timezone like cronv1

### PR DESCRIPTION
**Pull request by archseer**

### CODE REVIEW

File: core/services/cron/validate.go [hunk 1]
```diff
@@ -36,6 +38,10 @@ func ValidateCronSpec(tomlString string) (job.Job, error) {
 		return jb, errors.Errorf("the only supported schema version is currently 1, got %v", jb.SchemaVersion)
 	}
 
+	if !(strings.HasPrefix(spec.CronSchedule, "CRON_TZ=") || strings.HasPrefix(spec.CronSchedule, "@")) {
```
Comment (samsondav) [inline core/services/cron/validate.go:41]: Is this the same as what we already do in cronv1? I vaguely recall that the library did the validation for us

File: core/services/cron/validate.go [hunk 2]
```diff
@@ -36,6 +38,10 @@ func ValidateCronSpec(tomlString string) (job.Job, error) {
...
+		return jb, errors.New("cron schedule must specify a time zone using CRON_TZ, e.g. 'CRON_TZ=UTC 5 * * * *'")
```
Comment (samsondav) [inline core/services/cron/validate.go:42]: Maybe document the `@every` format in this error message?

File: core/services/cron/validate.go (hunk 1 above)
Comment (archseer) [reply core/services/cron/validate.go:41]: Yeah I copied this from here: https://github.com/smartcontractkit/chainlink/blob/f78e43e394e88ddc5716cf61706ba3282ecf2ca9/core/store/models/common.go#L495-L520

The library will just default to localtime.

I kinda like the `Cron` type, we could reuse it for v2 by changing this field to use `Cron`? https://github.com/smartcontractkit/chainlink/blob/9e7bde8f3ab7dc2eabcf72cef5e6f656d0001e16/core/web/presenters/job.go#L175

Comment (archseer) [reply core/services/cron/validate.go:41]: Okay, `toml` doesn't support `UnmarshalTOML` so this wasn't doable

File: core/services/cron/validate.go (hunk 2 above)
Comment (archseer) [reply core/services/cron/validate.go]: Done!

**Review by samsondav — APPROVED** [review]

**Pull request merged**